### PR TITLE
search course

### PR DESCRIPTION
**Pull request by tranduytung**

### CODE REVIEW

**Comment by nguyenngocha** [comment]
OK

File: app/controllers/searchs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class SearchsController < ApplicationController
+  authorize_resource class: Course.name
+  def index
```
Comment (hellovietnam93) [inline app/controllers/searchs_controller.rb]: cashc dòng ra nhé

File: app/views/user_courses/_search.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <%= f.date_field :created_at_lteq, class: "radius" %>
+  </div>
+  <hr>
+  <div class="large-5 small-4">
+    <%= f.label :subjects_content_cont, t("search.courses.subject_content") %>
+    <%= f.search_field :subjects_content_cont %>
+  </div>
+  <hr>
+  <div class="column">
+    <%= f.submit t("search.courses.search_course"), class: " radius button" %>
```
Comment (nguyenngocha) [inline app/views/user_courses/_search.html.erb]: "radius button"

**Comment by nguyenngocha** [comment]
OK

**Comment by nhunght** [comment]
OK

**Pull request merged**